dlc-manager/tests/ln_dlc_channel_execution_tests.rs

```diff
@@ -1604,6 +1616,190 @@ fn ln_dlc_close_finalized_force_close() {
+
+    assert_sub_channel_state!(
+        test_params.alice_node.sub_channel_manager,
+        &test_params.channel_id,
+        Offered
+    );
+
+    assert_sub_channel_state!(
+        test_params.bob_node.sub_channel_manager,
+        &test_params.channel_id,
+        Accepted
+    );
+
+    let sub_channel = test_params
+        .bob_node
+        .dlc_manager
+        .get_store()
+        .get_sub_channel(test_params.channel_id)
+        .unwrap()
+        .unwrap();
+    let commit_tx = if let SubChannelState::Accepted(a) = &sub_channel.state {
+        a.commitment_transactions[0].clone()
+    } else {
+        unreachable!();
+    };
+
+    ldk_auto_close(&mut test_params, &commit_tx);
+}
+
+//TODO(tibo): Find a way to trigger an ldk auto force close in this state. Changing the fee does
+//not work because Alice is awaiting a revoke from Bob and doesn't send the fee update message,
+//sending a bad commitment signature also doesn't really fit with the state.
+// fn ln_dlc_ldk_auto_close_confirmed_test() {
+// }
+// #[test]
+// #[ignore]
+// fn ln_dlc_ldk_auto_close_finalized_test() {
+// }
+
+#[test]
+#[ignore]
+fn ln_dlc_ldk_auto_close_close_offered_test() {
+    let mut test_params = test_init();
+
+    make_ln_payment(&test_params.alice_node, &test_params.bob_node, 900000);
+
+    open_sub_channel(&test_params);
+
+    go_to_off_chain_close_state(&test_params, TargetState::OfferReceived, false);
+
+    assert_sub_channel_state!(
+        test_params.alice_node.sub_channel_manager,
+        &test_params.channel_id,
+        CloseOffered
+    );
+
+    assert_sub_channel_state!(
+        test_params.bob_node.sub_channel_manager,
+        &test_params.channel_id,
+        CloseOffered
+    );
+
+    mocks::mock_time::set_time(EVENT_MATURITY as u64);
+
+    let commit_tx =
+        get_commit_tx_from_node(&test_params.bob_node, &test_params.funding_txo).remove(0);
+
+    ldk_auto_close(&mut test_params, &commit_tx);
+}
+
+#[test]
+#[ignore]
+fn ln_dlc_ldk_auto_close_close_accepted_test() {
+    let mut test_params = test_init();
+
+    make_ln_payment(&test_params.alice_node, &test_params.bob_node, 900000);
+
+    open_sub_channel(&test_params);
+
+    go_to_off_chain_close_state(&test_params, TargetState::Accepted, false);
+
+    assert_sub_channel_state!(
+        test_params.alice_node.sub_channel_manager,
+        &test_params.channel_id,
+        CloseOffered
+    );
+
+    assert_sub_channel_state!(
+        test_params.bob_node.sub_channel_manager,
+        &test_params.channel_id,
+        CloseAccepted
+    );
+
+    mocks::mock_time::set_time(EVENT_MATURITY as u64);
+
+    let commit_tx =
+        get_commit_tx_from_node(&test_params.bob_node, &test_params.funding_txo).remove(0);
+
+    ldk_auto_close(&mut test_params, &commit_tx);
+}
+
+//TODO(tibo): find a way to test this state.
+// #[test]
+// #[ignore]
+// fn ln_dlc_ldk_auto_close_close_confirmed_test() {
+// }
```

Comment:
For all the states for which force-close cannot be triggered with the fee estimate change, I would just call force-close on the LN channel directly?

Reply:
I was hesitating to do that because it's somehow not something that you expect from proper usage. But well it's easier at the end so I just use that for all the tests now :)